src/cli/java/org/commcare/util/screen/QueryScreen.java

```diff
@@ -146,23 +152,31 @@ public String buildUrl(boolean skipDefaultPromptValues) {
 
 
     private InputStream makeQueryRequestReturnStream() {
-        String url = buildUrl(false);
+        queryUrl = buildUrl(false);
         String credential = Credentials.basic(domainedUsername, password);
 
         Request request = new Request.Builder()
-                .url(url)
+                .url(queryUrl)
                 .header("Authorization", credential)
                 .build();
         return client.makeRequest(request);
     }
 
-    public Pair<ExternalDataInstance, String> processResponse(InputStream responseData) {
+    public Pair<ExternalDataInstance, String> processResponse(InputStream responseData, String url) {
```

Comment:
in the formplayer PR it seems this method call was replaced with something else (https://github.com/dimagi/formplayer/pull/1006#discussion_r725955421). Is this method still needed?